src/components/Icon/DotsIcon/DotsIcon.jsx

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass, omitProps } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-DotsIcon');
+
+const DotsIcon = createClass({
+	displayName: 'DotsIcon',
+
+	statics: {
+		peek: {
+			description: `
+				Three dots in a row.
+			`,
+			categories: ['visual design', 'icons'],
+			extend: 'Icon',
+			madeFrom: ['Icon'],
+		},
+	},
+
+	propTypes: {
+		...Icon.propTypes,
+	},
+
+	render() {
+		const { className, ...passThroughs } = this.props;
+
+		return (
+			<Icon
+				{...omitProps(passThroughs, DotsIcon, [], false)}
+				{..._.pick(passThroughs, _.keys(Icon.propTypes))}
+				className={cx('&', className)}
+			>
+				<circle cx='8' cy='8' r='1' />
```

Comment:
As mentioned in the ticket, I think we need to also fill these circles which makes this icon a little bit of a snowflake. We could make the fill support the prop variations that @astropunk added at least.

Reply:
Done